cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -255,6 +255,16 @@ rules:
   - network-attachment-definitions
   verbs:
   - get
+- apiGroups:
+  - route.openshift.io
+  resources:
+  - routes
+  verbs:
+  - get
+  - create
+  - update
+  - list
```

Comment:
No need for `list` or `watch`, unless you find a simple way to only watch on openshift.